infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -256,6 +267,13 @@ private TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
                         break;
                     }
 
+                    if (txSnapshotTimestamp < maxConflictNewSequencer) {
```

Comment:
Can you add a brief comment here explaining when is this `maxConflictNewSequencer` modified and why are we aborting this?

Reply:
Done